projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -253,6 +253,7 @@ <h3 id="{{ 'doc-altgr-title-' + i }}">{{ altgr_title }}</h3>
         *ngIf="!record.metadata.harvested && record.metadata.pid"
         id="documents-local-field-tab"
         tabOrder="4"
+        [customClass]="showhideLocalFieldsTab"
```

Comment:
why not use this condition into the `*ngIf` ?
Additionally the name of the function could be more elegant and return a boolean value